devportal/devportal-domain-service/ballerina/resources/devportal-api.yaml

```diff
@@ -5547,7 +5501,7 @@ components:
       name: X-WSO2-Tenant
       in: header
       description: |
-        For cross-tenant invocations, this is used to specify the tenant domain, where the resource need to be
+        For cross-tenant invocations, this is used to specify the /organization domain, where the resource need to be
```

Comment:
@sanjeewa-malalgoda , looks like we do not need `/` before the word "organization".

Reply:
fixed